key.js

```diff
@@ -122,15 +52,13 @@ const gameModule = (function () {
     }
     function resetGameState() {
         gameState = getInitialGameState();
```

Comment:
This won't work, since re-assigning to a `const` is not possible, but you can either do the following:

1. Make `gameState` a `let` object: `let gameState = getInitialGameState();`, and then this line is valid. (_OR_)
2. Keep the `gameState` as a `const` object, but write a method to `resetGameState`, as follows:
    ```js
    function resetGameState() {
      const initialGameState = getInitialGameState();
      Object.keys(gameState).forEach(gameStateKey => gameState[key] = initialGameState[key]);
    }
    ```